integration-tests/tests/create-application-from-sample.spec.ts

```diff
@@ -6,17 +6,27 @@ import { Applications } from '../utils/Applications';
 import { Common } from '../utils/Common';
 
 describe('Create Application from Sample', () => {
+  const LOCAL_STORAGE_KEY_GS_MODAL = 'getting-started-modal';
+  const LOCAL_STORAGE_KEY_APPLICATION_MODAL = 'showApplicationModal';
   const applicationName = Common.generateAppName();
   const applicationDetailPage = new ApplicationDetailPage();
   const componentPage = new ComponentPage();
   const addComponent = new AddComponentPage();
   const componentSamplesPage = new ComponentSamplesPage();
+  const publicRepos = [
+    'https://github.com/nodeshift-starters/devfile-sample.git',
+    'https://github.com/devfile-samples/devfile-sample-code-with-quarkus.git'
+  ];
 
-  before(() => {
-    // Disable HACBS
-    localStorage.setItem('hacbs', 'false');
-    // Need to reload the page after enabling HACBS via localStorage
-    cy.reload();
+  beforeEach(function () {
+    localStorage.setItem(LOCAL_STORAGE_KEY_GS_MODAL, 'true');
+    localStorage.setItem(LOCAL_STORAGE_KEY_APPLICATION_MODAL, 'true');
```

Comment:
I see you set those in every test. What about setting it up once there https://github.com/dheerajodha/hac-dev/blob/fix-hacdev-tests/integration-tests/support/commands/hooks.ts#L15 ? It should work for all tests then.

Reply:
Done, thank you!